### PR TITLE
Removed schemes leftovers that are no longer needed

### PR DESCRIPTION
**Pull request by kobizz**
## PR Checklist

- [ ] The commit message follows our guidelines:  https://github.com/elementor/elementor/blob/master/.github/CONTRIBUTING.md

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] Other... Please describe:

## Summary

This PR can be summarized in the following changelog entry:

*

## Description
An explanation of what is done in this PR

*

## Test instructions
This PR can be tested by following these steps:

*

## Quality assurance

- [ ] I have tested this code to the best of my abilities
- [ ] I have added unittests to verify the code works as intended
- [ ] Docs have been added / updated (for bug fixes / features)

Fixes #

### CODE REVIEW

File: core/files/css/global-css.php [hunk 1]
```diff
@@ -93,21 +92,7 @@ protected function get_inline_dependency() {
...
-		}
-
-		$elementor_settings_last_updated = get_option( Settings::UPDATE_TIME_FIELD );
-
-		if ( $file_last_updated < $elementor_settings_last_updated ) {
-			return true;
-		}
-
-		return false;
+		return $this->get_meta( 'time' ) < get_option( Settings::UPDATE_TIME_FIELD );
```
Comment (matipojo) [inline core/files/css/global-css.php:95]: On a refresh install both values are empty, and it will return false even the update is required.

**Review by matipojo — CHANGES_REQUESTED** [review]

File: core/files/css/global-css.php (hunk 1 above)
Comment (kobizz) [reply core/files/css/global-css.php:95]: It's exactly like the previous check. Only the schemes file check was removed

Comment (matipojo) [reply core/files/css/global-css.php:95]: Currently, on a fresh install it will return false.

Comment (kobizz) [reply core/files/css/global-css.php:95]: Can you provide me with a real case (with steps to reproduce) when you encounter a problem as a result of this behavior?

Comment (matipojo) [reply core/files/css/global-css.php:95]: OK, I checked and a fresh install is covered by another logic.

**Review by matipojo — APPROVED** [review]

**Review by KingYes — APPROVED** [review]

**Pull request merged**